examples/jsm/controls/OrbitControls.js

```diff
@@ -292,17 +288,35 @@ class OrbitControls extends EventDispatcher {
 
 		}();
 
+		this.connect = function (domElement) {
+
+			if ( domElement === document ) console.error( 'THREE.OrbitControls: "document" should not be used as the target "domElement". Please use "renderer.domElement" instead.' );
+
+			scope.domElement = domElement;
+			scope.domElement.style.touchAction = 'none'; // disable touch scroll
+
+			scope.domElement.addEventListener( 'contextmenu', onContextMenu );
+
+			scope.domElement.addEventListener( 'pointerdown', onPointerDown );
+			scope.domElement.addEventListener( 'pointercancel', onPointerCancel );
+			scope.domElement.addEventListener( 'wheel', onMouseWheel, { passive: false } );
+
+		};
+
 		this.dispose = function () {
 
-			scope.domElement.removeEventListener( 'contextmenu', onContextMenu );
```

Comment:
I suggest to rename `dispose()` to `disconnect()`. You can then implement a new `dispose()` method to ensure backwards compatibility like this:
```js
this.dispose = function () {

    scope.disconnect();
    console.warn( 'THREE.OrbitControls: dispose() has been deprecated. Use disconnect() instead.' );

};

Reply:
Maybe it would be better to remove this warning? Other controls also have `dispose` methods, and if we will use the connect/disconnect approach there, some `dispose`s would do something more than just `disconnect`, so it would be better to have both methods available.

Reply:
`dispose()` and `disconnect()` are semantically the same thing. I don't think it makes sense to have both of them.

Reply:
`PointerLockControls` have `connect` + `disconnect` and `dispose` (without warning)

Reply:
Agreed! Let's keep `dispose()`. The distinction between `dispose()` and `disconnect()` does make sense with at least `TransformControls`.